indico/modules/events/contributions/models/contributions.py

```diff
@@ -247,6 +249,22 @@ def track(self):
     def person_link_data(self):
         return {x: x.is_submitter for x in self.person_links}
 
+    @property
+    def authors(self):
+        return {x for x in self.person_links if x.author_type == AuthorType.primary}
+
+    @property
+    def coauthors(self):
+        return {x for x in self.person_links if x.author_type == AuthorType.secondary}
+
+    @property
+    def speakers(self):
+        return {x for x in self.person_links if x.is_speaker}
+
+    @property
+    def attached_items(self):
```

Comment:
I think @pferreir  already added  this through a mixin in his PR (not merged yet)